src/test/java/seedu/rc4hdb/logic/commands/misccommands/MiscCommandTestUtil.java

```diff
@@ -0,0 +1,36 @@
+package seedu.rc4hdb.logic.commands.misccommands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import seedu.rc4hdb.logic.commands.CommandResult;
+import seedu.rc4hdb.logic.commands.exceptions.CommandException;
+
+/**
+ * Contains helper functions for testing {@code MiscCommand}.
+ */
+public class MiscCommandTestUtil {
+
+    /**
+     * Executes the given {@code command}, confirms that <br>
+     * - the returned {@link CommandResult} matches {@code expectedCommandResult} <br>
+     * - the {@code actualModel} matches {@code expectedModel}
+     */
+    public static void assertCommandSuccess(MiscCommand command, CommandResult expectedCommandResult) {
```

Comment:
Good idea to create the test classes and methods beforehand even if we're not using them now. I think this is why the code coverage has been reduced by a bit but it's fine for now since we haven't utilized the MiscCommand Class anywhere.